bin/ExtractGName.R

```diff
@@ -7,7 +10,7 @@ zip.fname <- paste(fname, 'zip', sep='.')
 heatmap.dat <- file.path(fname, 'heatmap.RData')
 load(unz(zip.fname, heatmap.dat))
 
-gene.list <- strsplit(go.list[[1]], ':')
-gname.list <- sapply(gene.list, function(x) x[1])
+gene.list <- as.data.frame(go.list, stringsAsFactors = FALSE)
+gname.list <- str_split_fixed(gene.list[,1],':',2)[,1]
```

Comment:
I've added a new commit which outputs a table with gene name , Ensembl ID and clusters
It also spits out files for each cluster containing gene names. (I've been piping these files back into ngsplot to create individual heatmaps for each cluster)

Still need to make it more compatible with the other algorithms and with multi plots